README.md

```diff
@@ -1 +1,45 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Linters installation.
```

Comment:
- [ ] Great job mentioning the title of your project. But it would be better to add a description of your project to your README. I would recommend the following structure:
```md
# Hello_microverse
> Hello_microverse is ...
```